utils/pinprocfw/pinprocfw.cpp

```diff
@@ -2134,13 +2136,10 @@ int processFile()
             endClock    = clock();
             fclose( in );
 
-            // Destroy the P-ROC device handle:
-            PRDelete(proc);
-            proc = kPRHandleInvalid;
             return iErrorCode;
 }
 
-int checkPROCFile() {
+uint32_t checkPROCFile() {
```

Comment:
In some of my builds I was getting a compiler warning related to the `switch` statement below.  Since we're returning a `uint32` variable in the success path, it makes sense to update the API to return that data type.